modules/gitops-argocd-cli-creating-an-application-in-core-mode.adoc

```diff
@@ -0,0 +1,142 @@
+endif::cluster[]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI. 
+
+.Procedure
+
+. Log in to the {OCP} cluster using the `oc` CLI tool:
++
+[source,terminal]
+----
+$ oc login -u <username> -p <password> <server_url>
+----
++
+.Example
+[source,terminal]
+----
+$ oc login -u kubeadmin -p '<password>' https://api.crc.testing:6443
+----
+
+. Check whether the context is set correctly in the `kubeconfig` file:
++
+[source,terminal]
+----
+$ oc config current-context
+----
+
+. Set the default namespace of the current context to `openshift-gitops`:
++
+[source,terminal]
+----
+$ oc config set-context --current --namespace openshift-gitops
+----
+
+. Set the following environment variables to override the Argo CD component names:
++
+[source,terminal]
+----
+$ export ARGOCD_REPO_SERVER_NAME=openshift-gitops-repo-server
+----
+
+. Verify that you are able to run `argocd` commands in `core` mode by listing all applications:
++
+[source,terminal]
+----
+$ argocd app list --core
+----
++
+If the configuration is correct, then existing applications will be listed with the following header:
++
+.Sample output
+[source,terminal]
+----
+NAME CLUSTER NAMESPACE  PROJECT  STATUS  HEALTH   SYNCPOLICY  CONDITIONS  REPO PATH TARGET
+----
+
+ifdef::cluster[]
+. Create an application in `core` mode:
++
+[source,terminal]
+----
+$ argocd app create app-cluster-configs --core \
+    --repo https://github.com/redhat-developer/openshift-gitops-getting-started.git \
+    --path cluster \
+    --revision main \
+    --dest-server  https://kubernetes.default.svc \
+    --dest-namespace spring-petclinic \
+    --directory-recurse \
+    --sync-policy none \
+    --sync-option Prune=true \
+    --sync-option CreateNamespace=true
+----
+endif::cluster[]
+
+ifdef::app[]
+. Create an application in `core` mode:
++
+[source,terminal]
+----
+$ argocd app create app-spring-petclinic --core \
+    --repo https://github.com/redhat-developer/openshift-gitops-getting-started.git \
+    --path app \
+    --revision main \
+    --dest-server  https://kubernetes.default.svc \
+    --dest-namespace spring-petclinic \
+    --directory-recurse \
+    --sync-policy automated \
+    --self-heal \
+    --sync-option Prune=true \
+    --sync-option CreateNamespace=true
+----
+endif::app[]
+
+. Label the `spring-petclinic` destination namespace to be managed by the `openshif-gitops` Argo CD instance:
++
+[source,terminal]
+----
+$ oc label ns spring-petclinic "argocd.argoproj.io/managed-by=openshift-gitops"
+----
+
+ifdef::cluster[]
+. List the application to confirm that the application is created successfully: 
```

Comment:
Does this command list all of the available applications? If so, the beginning of this sentence is a little misleading.